src/manage_sql_nvts_common.c

```diff
@@ -0,0 +1,1102 @@
+/* Copyright (C) 2019-2025 Greenbone AG
+*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * @file manage_sql_nvts_common.c
+ * @brief GVM management layer: Common NVT logic
+ *
+ * Shared NVT logic for the GVM management layer.
+ */
+
+#include "iterator.h"
+#include "manage_sql.h"
+#include "manage_sql_configs.h"
+#include "manage_sql_nvts_common.h"
+#include "sql.h"
+
+#include <gvm/base/cvss.h>
```

Comment:
I think we need to add the log domain to the new files. What do you think ?
```suggestion
#include <gvm/base/cvss.h>

#undef G_LOG_DOMAIN
/**
 * @brief GLib log domain.
 */
#define G_LOG_DOMAIN "md manage"
```